app/services/bestuursorganen.js

```diff
@@ -0,0 +1,62 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+
+export default class BestuursorganenService extends Service {
+  @service store;
+  @service decretaleOrganen;
+
+  async getAllRealPoliticalBestuursorganen() {
+    return await this.store.query('bestuursorgaan', {
+      'filter[:has-no:deactivated-at]': true,
+      'filter[:has-no:is-tijdsspecialisatie-van]': true,
+      'filter[:has-no:original-bestuurseenheid]': true,
+      'filter[classificatie][id]':
+        this.decretaleOrganen.classificatieIds.join(','), // only organs with a political mandate
+      include: 'classificatie,heeft-tijdsspecialisaties',
+    });
+  }
+
+  async getFilteredRealPoliticalBestuursorganen(queryParams, bestuursperiode) {
+    const queryOptions = {
+      sort: queryParams.sort,
+      page: {
+        size: this.pageSize,
+      },
+      'filter[:has-no:is-tijdsspecialisatie-van]': true,
+      'filter[:has-no:original-bestuurseenheid]': true,
+      'filter[heeft-tijdsspecialisaties][heeft-bestuursperiode][:id:]':
+        bestuursperiode.id,
+      include: 'classificatie,heeft-tijdsspecialisaties',
+    };
+    if (queryParams.activeOrgans) {
+      queryOptions['filter[:has-no:deactivated-at]'] = true;
+    }
+    const types = queryParams.selectedTypes.map((type) => {
+      return this.decretaleOrganen.get(type).join(',');
+    });
+    queryOptions['filter[classificatie][:id:]'] = types.join(',');
+    return await this.store.query('bestuursorgaan', queryOptions);
+  }
+
+  async getAllPoliticalBestuursorganen() {
+    return await this.store.query('bestuursorgaan', {
+      'filter[:has-no:deactivated-at]': true,
+      'filter[:has-no:is-tijdsspecialisatie-van]': true,
+      'filter[classificatie][id]':
+        this.decretaleOrganen.classificatieIds.join(','), // only organs with a political mandate
+      include: 'classificatie,heeft-tijdsspecialisaties',
```

Comment:
Im also a fan of using .join(',') here for the includes. It gives a better overview IMO

Reply:
there is no reason to do this though, we're not getting any additional safety on the relation names or anything so directly using a string is just as fine imo